MC/config/PWGDQ/external/generator/GeneratorHFToMu_EvtGenFwdY_gaptriggered_dq.C

```diff
@@ -0,0 +1,203 @@
+#include "FairGenerator.h"
+#include "Generators/GeneratorPythia8.h"
+#include "Pythia8/Pythia.h"
+#include "TRandom.h"
+
+R__ADD_INCLUDE_PATH($O2DPG_ROOT/MC/config/PWGDQ/EvtGen)
+#include "GeneratorEvtGen.C"
+
+#include <string>
+
+using namespace o2::eventgen;
+
+namespace o2
+{
+namespace eventgen
+{
+
+class GeneratorHFToMu_EvtGenFwdY_gaptriggered : public o2::eventgen::GeneratorPythia8 {
+public:
+  
+  /// constructor
+  GeneratorHFToMu_EvtGenFwdY_gaptriggered(int inputTriggerRatio = 4)  {
+
+    mGeneratedEvents = 0;
+    mInverseTriggerRatio = inputTriggerRatio; 
+    // define minimum bias event generator
+    auto seed = (gRandom->TRandom::GetSeed() % 900000000);
+    TString pathconfigMB = gSystem->ExpandPathName("$O2DPG_ROOT/MC/config/PWGDQ/pythia8/generator/pythia8_inel_triggerGap.cfg");
```

Comment:
generator configs should no longer refer to `${O2DPG_ROOT}` but to `${O2DPG_MC_CONFIG_ROOT}`. Could you please fix this in all your files?

Reply:
Thank you. I changed all ${O2DPG_ROOT} in this updates to ${O2DPG_MC_CONFIG_ROOT}.